static/js/brand-store/Settings/Settings.tsx

```diff
@@ -19,30 +19,58 @@ import NotAuthorized from "../NotAuthorized";
 import PasswordToggle from "../shared/PasswordToggle";
 import SectionNav from "../SectionNav";
 
+declare global {
```

Comment:
These types of things I like to move to a shared global definition (`d.ts`) file as it'll be valid for the whole application.